### PR TITLE
Add uptime program

### PR DESCRIPTION
**Pull request by Wilma456**
This program use the new os.clock() function to show, how long the computer run.

### CODE REVIEW

File: src/main/resources/assets/computercraft/lua/rom/help/uptime.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+uptime prints how long your computer run
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/help/uptime.txt]: how long your computer has been running

File: src/main/resources/assets/computercraft/lua/rom/programs/uptime.lua [hunk 2]
```diff
@@ -0,0 +1,8 @@
+local function formatUptime( nTime )
+    local nHour = math.floor( nTime/60/60 )
+    local nMinute = math.floor( (nTime/60)  ) - nHour*60
+    local nSecond = math.floor( nTime ) - nMinute*60 - nHour*60*60
+    return nHour..":"..nMinute..":"..nSecond
+end
+
+print( "The Computer runs "..formatUptime( os.clock() ) )
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/programs/uptime.lua]: The computer has been running for ... hours.

File: src/main/resources/assets/computercraft/lua/rom/programs/uptime.lua [hunk 3]
```diff
@@ -0,0 +1,8 @@
+local function formatUptime( nTime )
+    local nHour = math.floor( nTime/60/60 )
+    local nMinute = math.floor( (nTime/60)  ) - nHour*60
+    local nSecond = math.floor( nTime ) - nMinute*60 - nHour*60*60
+    return nHour..":"..nMinute..":"..nSecond
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/programs/uptime.lua]: This formats time like `0:1:35`, when intuitively, it should be like `00:01:35`.

Comment (apemanzilla) [inline src/main/resources/assets/computercraft/lua/rom/programs/uptime.lua]: `string.format` would be able to do this much more concisely as well.

**Comment by dan200** [comment]
Doesn't seem necessary and not well written.

**Pull request closed without merging**